stdlib/public/Concurrency/Task.swift

```diff
@@ -618,26 +639,26 @@ public struct UnsafeCurrentTask {
     self._task = task
   }
 
-  /// Returns `Task` representing the same asynchronous context as this 'UnsafeCurrentTask'.
-  ///
-  /// Operations on `Task` (unlike `UnsafeCurrentTask`) are safe to be called
-  /// from any other task (or thread).
+  /// The current task,
+  /// represented in a way that's safe to store for later use.
```

Comment:
I much preferred the wording in terms of the operations that are safe to be called from any task, without necessarily implying storage.

Could we revive that?